core/tests/ts-integration/tests/api/web3.test.ts

```diff
@@ -808,6 +808,18 @@ describe('web3 API compatibility tests', () => {
         ).toBeRejected();
     });
 
+    test('Should check v value of the legacy transaction', async () => {
+        const LEGACY_TX_TYPE = 0;
+        let tx = await alice.sendTransaction({
+            type: LEGACY_TX_TYPE,
+            to: alice.address
+        });
+        let receipt = await tx.wait();
+
+        let block_with_transactions = await alice.provider.send('eth_getBlockByHash', [receipt.blockHash, true]);
+        expect(block_with_transactions.transactions[0].v >= 27);
```

Comment:
Can we somehow check the signature? like it was in the initial problem.

Reply:
Not sure what you mean by check the signature, the problem was the at `v` was in the range of [0,1]

Reply:
not sure

Reply:
I have run locally the script, that Chainlink uses. It does not return null, as it did before.

Reply:
can we integrate this script into ci?

Reply:
its Go

Reply:
so, I mean, a few commits above you messed up a bit with v,r,s positions and this test won't catch it. 

We have a very specific problem our signature was incorrect. That's why I want to check the signature correctness instead of checking some internal components, that we changed. 

Even more, it's an integration test. So you have to check the whole behavior as a black box, without knowing the exact issue. In this test, you are checking the exact params you changed. But it's better to check the desired behaviour and desired behaviour we can describe in two ways: 
1. The return tx is the exact tx we sent to the server 
2. The signature of this is correct